app/instance-initializers/configure-trackjs.js

```diff
@@ -18,6 +18,9 @@ export function initialize(app) {
   const handler = new ErrorHandler(trackJs);
 
   Ember.onerror = handler.report.bind(handler);
+  window.addEventListener('unhandledrejection', (event) => {
```

Comment:
Would you mind switching the function declaration style to `function(event) {`? This keeps this change in line with the rest of the code's style.

Is it possible to mock or trigger this event in a test to ensure this hook works?

Many thanks for the pull request!